server.py

```diff
@@ -234,26 +243,32 @@ def sendPart(self):
     return web.Response(status=204)
 
 @routes.post('/join')
-async def channeljoin(request):
+@routes.post('/part')
+async def channel_join_part(request):
```

Comment:
I guess these could be combined, though checking `request.path` twice worries me. There's probably a better way to organize this though.